### DISCUSSION TITLE
Has anyone experimented with removing book odds, changing loss function to decorrelate with book odds??

### DISCUSSION THREAD

**Discussion by notoctosting** [kyleskom/NBA-Machine-Learning-Sports-Betting#396] [Ideas]
bookmakers have pretty solid algorithms and their accuracy is hard to beat (unless they adjust them to balance out their accounts), and this model ends up significantly relying on the odds. Has anyone experimented with removing these features, or using a custom loss function to decorrelate the results from the odds? i.e. if models predictions are too similar to the books, we aren't making much money for the risk.

**Comment by kyleskom** [comment]
So here's the thing, books don't predict the outcomes of games. They predict what line / odds will get 50/50 split of the money on each side. Books make money when money is split evenly on the outcome.

**Comment by nealmick** [comment]
I have tried models with and without vegas odds and have found it be one of the largest changes in model behavior, models with odds tend to copy them.  The wisdom of the crowd is quite strong, most data features are priced into the market.  Its difficult to outperform vegas odds with stats.  Also this can be a scale problem, if the model is closely following vegas line maybe metrics need to be adjusted to capture smaller difference in projection.    It is also possible to compare different odds and create an average or fair price then look for bookies with lines that differ compared to the fair market odds.  Basically using aggregate market to find miss priced opportunities or bets that offer significant value.
